setup.py

```diff
@@ -28,26 +28,85 @@ def _zivid_python_version():
     return version
 
 
+def _make_message_box(*message_lines):
+    from tabulate import tabulate
+
+    return "\n\n" + tabulate([["\n\n".join(message_lines)]], tablefmt="grid") + "\n"
+
+
 def _check_dependency(module_name, package_hint=None):
     if package_hint is None:
         package_hint = module_name
     if module_name not in [module[1] for module in iter_modules()]:
         raise ImportError(
-            "Missing module '{}'. Please install '{}' manually or use PIP>=19 to handle build dependencies automatically (PEP 517).".format(
-                module_name, package_hint
+            _make_message_box(
+                "!! Missing module '{}' !!".format(module_name),
+                "Please install '{}' manually or use PIP>=19 to handle build dependencies automatically (PEP 517).".format(
+                    package_hint
+                ),
             )
         )
 
 
+def _check_cpp17_compiler():
```

Comment:
Doesn't seem to work. I deleted GCC and tried to build. 
Got the old and obscure error message.

Reply:
Seems to work fine in a Docker. Ignore this for now :)